commit 920ef55334b5043762de2667c41ac6072fe1ec92 (refs/changes/02/4602/2)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2018-07-30T17:00:25+00:00 (1 year, 2 months ago)
    
    joyent/gerritbot#5 disable check for repository "joyent/postgres"
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/server.js b/server.js
index c3a8003..86f2bc9 100644
--- a/server.js
+++ b/server.js
@@ -36,6 +36,7 @@ repoHasMakeCheck['joyent/illumos-extra'] = false;
 repoHasMakeCheck['joyent/illumos-joyent'] = false;
 repoHasMakeCheck['joyent/zfs_snapshot_tar'] = false;
 repoHasMakeCheck['joyent/illumos-kvm'] = false;
+repoHasMakeCheck['joyent/postgres'] = false;
 
 var dockerKeyPem = mod_fs.readFileSync(config.docker.keyFile);
 var dockerKey = mod_sshpk.parsePrivateKey(dockerKeyPem);
